assets/embauche.html

```diff
@@ -50,7 +50,16 @@
 					<option>mois</option>
 					<option disabled>jour</option>
 					<option disabled>heure</option>
-				</select>.
+				</select>
+
+				sur la commune de
+				<input name="code_postal_entreprise" id="code_postal_entreprise" class="medium-width centered"
+				type="text" placeholder="code postal" pattern="\d{5}" title="Entrez le code postal de l'établissement où le salarié sera employé">
```

Comment:
OK I guess you're refering to the [autocomplete attribute](https://html.spec.whatwg.org/multipage/forms.html#attr-fe-autocomplete) :
```html
<input autocomplete="postal-code">
```